content/es/guides/directory-structure/pages.md

```diff
@@ -0,0 +1,359 @@
+    return { name: 'World' }
+  }
+```
+
+<base-alert type="next">
+
+Aprende mas de cómo funciona asyncData en nuestro capítulo de [Data Fetching](/guides/features/data-fetching#async-data)
+
+</base-alert>
+
+### fetch
+
+Cada vez que necesites buscar data asincrona, puedes usar `fetch`. Fetch se ejecuta en el lado del servidor cuando se muestran las rutas, y en el lado del cliente (client-side) cuando el usuario navega la aplicación.
+
+```html
+<script>
+  export default {
+    data() {
+      return {
+        posts: []
+      }
+    },
+    async fetch() {
+      this.posts = await fetch('https://api.nuxtjs.dev/posts').then(res =>
+        res.json()
+      )
+    }
+  }
+</script>
+```
+
+<base-alert type="next">
+
+Conoce más sobre cómo `fetch` funciona en nuestro capítulo de [Data Fetching](/guides/features/data-fetching)
+
+</base-alert>
+
+### head
+
+Puedes configurar un <meta> tag específico para la página actual. Nuxt.js usa `vue-meta` para actualizar el encabezado del documento y los `meta` attributes de tu aplicación.
+
+```js{}[pages/index.vue]
+export default {
+  head() {
+    // Configura los meta tags para esta página
+  }
+}
+```
+
+<base-alert type="next">
+
+Conoce mas en nuestro capítulo de [Meta Tags y SEO](/guides/features/meta-tags-seo).
+
+</base-alert>
+
+### layout
+
+Se usa para especificar un `layout` definido en el directorio de layouts.
+
+```js{}[pages/index.vue]
+export default {
+  layout: 'blog'
+}
+```
+
+<base-alert type="next">
+
+Conoce mas en nuestro capítulo de [Vistas](/guides/concepts/views#layouts).
+
+</base-alert>
+
+### loading
+
+Si esta propiedad tiene un valor de `false`, previene la página de ejecutar `this.$nuxt.$loading.finish()` automáticamente cuando entras a ella y `this.$nuxt.$loading.start()` cuando te vas. Esto te permite controlar el comportamiento de la página manualmente, como se muestra en [este ejemplo](https://nuxtjs.org/examples/custom-page-loading).
+
+```js{}[pages/index.vue]
+export default {
+  loading: false
+}
+```
+
+<base-alert type="info">
+
+Sólo aplica si `loading` está configurado también en el `nuxt.config.js`
+
+</base-alert>
+
+<base-alert type="next">
+
+Conoce más sobre esto en el capítulo de [Loading](/guides/features/loading).
+
+</base-alert>
+
+### transition
+
+La propiedad de transición define una transición específica para la página.
+
+```js{}[pages/index.vue]
+export default {
+  transition: 'fade'
+}
+```
+
+<base-alert type="next">
+
+Conoce más sobre transiciones en el capítulo de [Transiciones](/guides/features/transitions).
+
+</base-alert>
+
+### scrollToTop
+
+La propiedad de `scrollToTop` te permite decirle a Nuxt.js que se desplpaze hacia el tope de la página antes de que se muestre. Nuxt.js desplaza la página hacia el tope por defecto, pero en rutas internas, Nuxt.js mantiene la posición de desplazamiento (scroll position). Si deseas que Nuxt se desplaze al tope cuando muestre las rutas internas, configura `scrollToTop` a cierto (`true`).
```

Comment:
The word **desplpaze** is miswritten it should be "**desplace**". Also in the last sentence the word _desplaze_ is written with a z, when it should be a c instead, desplace. 

Just for reference on the conjugation of the [verb desplazar](https://dle.rae.es/desplazar)

Reply:
- [ ] fix